### DISCUSSION TITLE
Rebuild a MultiOutputRegressor from MultiOutputRegressor(AutoML())

### DISCUSSION THREAD

**Discussion by pohpas** [microsoft/FLAML#484] [Q&A]
Hi everyone,

I ran a session with sklearns MultiOutputRegressor with:
```
model = MultiOutputRegressor(AutoML(**settings))
model.fit(X_train, y_train)
```

However, after fitting, MultiOutputRegressor keeps AutoML as estimator. 
Does anybody know, how to rebuild a MultiOutputRegressor with the result of AutoML, e.g.:
`MultiOutputRegressor([LGBRegressor(best_config1), LGBRegressor(best_config2), LGBRegressor(best_config3),])`

Appreciate your help!

**Comment by sonichi** [comment]
The only way I can think of is to inherit MultiOutputRegressor and override its (fit)[https://github.com/scikit-learn/scikit-learn/blob/37ac6788c9504ee409b75e5e24ff7d86c90c2ffb/sklearn/multioutput.py#L153] function. Set `self.estimators_` to be the list of estimators you want, and modify https://github.com/scikit-learn/scikit-learn/blob/37ac6788c9504ee409b75e5e24ff7d86c90c2ffb/sklearn/multioutput.py#L202-L207

**Reply by pohpas** [reply]
Will try, thank you!

**Reply by sonichi** [reply]
Thanks. I'd appreciate your feedback in a survey: https://docs.google.com/forms/d/e/1FAIpQLSd0u4BbSzr-m9hgM8rmPlYB_JJoFXeHhuoXI9YFbHDRWrDlOQ/viewform?usp=sf_link.
